### PR TITLE
worked on testing and the Home page in order to test that

### PR DESCRIPTION
**Pull request by arnoldlaoang**

### CODE REVIEW

File: app/javascript/components/pages/Home.js [hunk 1]
```diff
@@ -1,12 +1,15 @@
 import React, { Component } from 'react'
+import playButton from '../assets/playButton.png'
 
 export class Home extends Component {
   render() {
     return (
       <div>
-        <h1>Home</h1>
+        <h1 id="home-header" className="index-name">
+          TIRED OF THE SAME OLD SONGS, LET'S VIBE OUT!</h1>
```
Comment (WalkerTexas-coder) [inline app/javascript/components/pages/Home.js:9]: This is good! drop the closing h1 down a line

File: spec/requests/songs_spec.rb [hunk 2]
```diff
@@ -31,8 +31,7 @@
         song_response = JSON.parse(response.body)
         p song_response
         expect(song_response["title"]).to eq "i miss the old Kayne" #didnt use the index [0] since it wasnt returning an array
-      end
-    end
+      endar
```
Comment (WalkerTexas-coder) [inline spec/requests/songs_spec.rb:34]: endar?

**Review by WalkerTexas-coder — COMMENTED** [review]
couple comments

File: app/javascript/components/App.css [hunk 3]
```diff
@@ -7,3 +7,15 @@
...
+  font-size: 100px;
+  text-align: center;
+}
+
+#play-image {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
+}
```
Comment (development-reviewed) [inline app/javascript/components/App.css:21]: Not going to block, but if you haven't already done so, please enable "End of Line" settings in VSCode.

File: app/javascript/components/pages/Home.test.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    );
+  })
+  
+  it("checks all image's props", () => {
+    const img = home.find("img")
+    expect(img.prop("src")).toEqual(playButton)
+    expect(img.prop("alt")).toEqual("playButton")
+    expect(img.prop("id:")).toEqual("play-image");
+  })
+})
```
Comment (development-reviewed) [inline app/javascript/components/pages/Home.test.js:31]: End of File

File: app/javascript/components/pages/SongEdit.test.js [hunk 5]
```diff
@@ -20,8 +21,13 @@ describe("When SongEdit renders", () => {
     expect(songEditHeading).toEqual("Update or Delete Song Information");
   });
 
+  it("displays a song form", () => {
+    const songForm = edit.find("Form");
+    expect(songForm.length).toEqual(1);
+  });
+
   xit("displays song title", () => {
```
Comment (development-reviewed) [inline app/javascript/components/pages/SongEdit.test.js:29]: I know we commented this out, but I encourage you to keep at it as you'll have a much easier time developing with good tests.

**Comment by VonTeacher** [comment]
Hi all,

I'm seeing the following errors that need to be resolved before this pull request can be accepted.

```bash
 FAIL  config/webpack/test.js
  ● Test suite failed to run

    Your test suite must contain at least one test.

      at onResult (node_modules/@jest/core/build/TestScheduler.js:172:18)
      at node_modules/@jest/core/build/TestScheduler.js:300:17
      at node_modules/emittery/index.js:311:13
          at Array.map (<anonymous>)
      at Emittery.emit (node_modules/emittery/index.js:309:23)

 FAIL  app/javascript/components/pages/Home.test.js
  ● Test suite failed to run

    Jest encountered an unexpected token

    Jest failed to parse a file. This happens e.g. when your code or its dependencies use non-standard JavaScript syntax, or when Jest is not configured to support such syntax.

    Out of the box Jest supports Babel, which will be used to transform your files into valid JS based on your Babel configuration.

    By default "node_modules" folder is ignored by transformers.

    Here's what you can do:
     • If you are trying to use ECMAScript Modules, see https://jestjs.io/docs/ecmascript-modules for how to enable it.
     • If you are trying to use TypeScript, see https://jestjs.io/docs/getting-started#using-typescript
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/configuration
    For information about custom transformations, see:
    https://jestjs.io/docs/code-transformation

    Details:

    /Users/Shared/Learn/StarTunes/app/javascript/components/assets/playButton.png:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,jest){�PNG


    SyntaxError: Invalid or unexpected token

      1 | import React, { Component } from 'react'
    > 2 | import playButton from '../assets/playButton.png'
        | ^
      3 |
      4 | export class Home extends Component {
      5 |   render() {

      at Runtime.createScriptFromCode (node_modules/jest-runtime/build/index.js:1773:14)
      at Object.<anonymous> (app/javascript/components/pages/Home.js:2:1)
```

If there is other work you want to get moving on before tests, then you'll need to close this PR and break it into small pull requests.

**Review by VonTeacher — APPROVED** [review]
Good to see the test coverage, though for future purposes, look into mocking/stubbing elements to avoid having to add lots of assets just for testing purposes.

**Pull request merged**